src/components/DeploymentOptionsGrid/DeploymentOptionsGrid.tsx

```diff
@@ -0,0 +1,116 @@
+import { Realm, signupLogic } from 'logic/signupLogic'
+import { useActions, useValues } from 'kea'
+
+const BenefitsList = ({ items }: { items: (string | React.ReactNode)[] }): JSX.Element => {
+    return (
+        <ul>
+            {items.map((child, i) => (
+                <li key={i}>
+                    {child}
+                    <span className="separator-line" />
+                </li>
+            ))}
+        </ul>
+    )
+}
+
+export const DeploymentOptionsGrid = (): JSX.Element => {
+    const { email } = useValues(signupLogic)
+    const { reportDeploymentTypeSelected } = useActions(signupLogic)
+    return (
+        <div className="w-full">
+            <div className="text-white text-center">
+                <h1 className="leading-tight mb-4">Choose your edition</h1>
+                <p className="opacity-75">Get the same product suite, whether you self-host or choose PostHog Cloud.</p>
+            </div>
+            <div className="grid grid-cols-2 gap-4 pricing-grid">
+                <div className="pricing-card">
+                    <h2 className="bg-yellow">
+                        <ServerIcon />
+                        Self-host
+                    </h2>
+                    <div className="card-body">
+                        <div className="icon-flex-row">
+                            <div className="icon-lockup">
+                                <BadgeWrapper
+                                    style={{
+                                        background: 'rgba(251, 79, 13, 0.1)',
+                                        color: '#F75A01',
+                                    }}
+                                >
+                                    <OpenSourceIcon />
+                                </BadgeWrapper>
+                                <h3 className="font-sans">Open source</h3>
+                                <p>Free forever</p>
+                            </div>
+                            <div className="icon-lockup">
+                                <BadgeWrapper
+                                    style={{
+                                        background: 'rgba(58, 94, 244, 0.1)',
+                                        color: '#3A5EF4',
+                                    }}
+                                >
+                                    <UpAndToTheRightIcon />
+                                </BadgeWrapper>
+                                <h3>Enterprise</h3>
+                                <p>From $2k/mo</p>
+                            </div>
+                        </div>
+                        <p>Benefits</p>
+                        <BenefitsList
+                            items={[
+                                'Data stays on your infrastructure',
+                                'Full access to production instance',
+                                'No third-party cookies',
```

Comment:
We should consider calling out "Ad blocker-resistant" as that seems to be a reason someone would choose to self host. Not sure if that's better than "No third-party cookies"

Reply:
I think no third-party cookies is a lot clearer and has the same effect for someone that understand how adblockers work - in my experience a lot of the time teams are hesitant to add any third party cookies, whether they are adblocked or not.

Reply:
You can make a third-party cookie ad-block resistant, but you can't remove third-party cookies altogether with a cloud solution

Reply:
I didn't see any mention of third-party cookies in the full comparison (pricing page), but it is highlighted in the short comparison above it.